wiki/release/release-process.md

```diff
@@ -467,11 +424,11 @@ cp pulsar-broker/target/docs/*.json site2/website/static/swagger/2.X.0
 ```
 Send out a PR request for review.
 
-## 16. Write release notes
+## Write release notes
 
-Steps and examples see [Pulsar Release Notes Guide](https://docs.google.com/document/d/1cwNkBefKyV6OPbEXnUrcCdVZi0i2BezqL6vAL7VqVC0/edit#).
+See [Pulsar Release Notes Guide](https://docs.google.com/document/d/1cwNkBefKyV6OPbEXnUrcCdVZi0i2BezqL6vAL7VqVC0/edit#).
```

Comment:
I hope we can inline this guide instead of holding a dangling Google Docs page. But it can be a follow-up.

Reply:
I agree. Actually I'm opposed to referencing an evolving document in the repo. Especially it's not a huge document. We usually use an external link when the content is huge, like the [Pulsar's transaction design](https://github.com/apache/pulsar/wiki/PIP-31%3A-Transaction-Support).

/cc @Anonymitaet